[ADD] accounting/l10n: new egyptian localization and einvoicing section

jcs-odoo opened the pull request:
https://www.odoo.com/web#id=2858349&menu_id=4720&cids=1&action=333&active_id=3835&model=project.task&view_type=form

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/2109).

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+.. note::
+   - ETA should give you a username and a password to access their online portal.
+   - Ask ETA to provide you with preproduction portal access as well.
+   - These codes are confidential and should be saved safely.
```
I might be mistaken, but perhaps consider using "should be stored safely" instead of "saved safely".

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+Customers
+~~~~~~~~~
+
+Make sure your customers' contact forms are correctly filled out so your e-invoices are valid:
```
In line 102 you spelled it eInvoicing, and on this line, you spelled it e-invoices with a hyphen.

jcs-odoo replied:
Yes, I thought about this too. I need to be consistent, but it seems that the ETA calls it "eInvoicing" and similar, while the main spelling in the dictionary (don't remember which one) spells it e-invoicing. I'll fix this, thanks :)

toaa-odoo commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+Products
+~~~~~~~~
+
+Make sure your products are correctly configured so your e-invoices are valid:
```
Again, e-invoices vs eInvoicing.

toaa-odoo reviewed (changes requested):
Looks good to me, but maybe check spelling for "eInvoicing" and "e-invoices" :)

jcs-odoo commented:
Salut @toaa-odoo  :)
I fixed the e-invoicing thing. I only left it at line 125 as "eInvoicing" on purpose, because that's how the title is written on that page.

I also fixed the capitalization of my headings...

You can see the changes since my last version by clicking on "compare" here
![image](https://user-images.githubusercontent.com/46810843/171462327-8132750d-bd9a-4f7d-b55c-3d09de91935c.png)

Thanks :)

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+   </administration/upgrade>` your database.
+
+.. seealso::
+   - `Video: Egypt E-invoicing <https://www.youtube.com/watch?v=NXuBPLR4pVw>`_
```
Is it intentional not to embed the video?

jcs-odoo replied:
It is indeed. I don't want the video to become the primary info here. But it is listed at the top of the page to make it easily reachable.

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+   - **Test on your preproduction portal** before starting to issue real invoices on the production
+     ETA portal.
+
+     - **Credentials** for preproduction and production environments are different. Make sure to
```
Looks like this line is over-indented?

jcs-odoo replied:
It wasn't but it can be up a level if you prefer.

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+
+- :guilabel:`Product Type`: storable products, consumables, or services.
+- :guilabel:`Unit of Measure`: if you also use Odoo Inventory and have enabled :doc:`Units of
+  Measure <../../../../inventory_and_mrp/inventory/management/products/uom>`
```
Since your relative path goes all the way back to the root of applications, you may want to do `/applications/...` instead.

jcs-odoo replied:
Ouais, je l'avais fait pour les chemins qui partent directement du root, mais okay :)

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+
+- :guilabel:`Product Type`: storable products, consumables, or services.
+- :guilabel:`Unit of Measure`: if you also use Odoo Inventory and have enabled :doc:`Units of
+  Measure <../../../../inventory_and_mrp/inventory/management/products/uom>`
```
```suggestion
  Measure <../../../../inventory_and_mrp/inventory/management/products/uom>`.
```

AntoineVDV commented on content/applications/finance/accounting/fiscal_localizations/localizations/egypt.rst:
```diff
@@ -0,0 +1,244 @@
+  Measure <../../../../inventory_and_mrp/inventory/management/products/uom>`
+- :guilabel:`Barcode`: **GS1** or **EGS** barcode
+
+  - :guilabel:`ETA Item code` (under the :menuselection:`Accounting` tab): if the barcode doesn't
```
Over-indented? Maybe not but I personally find it weird to use bullet points for additional details.

jcs-odoo replied:
I can put it up a level if you prefer. It doesn't matter much here indeed

AntoineVDV reviewed (approved):
@robodoo r+
